extensions/parrot-translate/src/parrot.tsx

```diff
@@ -0,0 +1,178 @@
+                                                <ListActionPanel
+                                                    queryText={inputState}
+                                                    copyMode={copyModeState}
+                                                    copyText={item?.subtitle || item.title}
+                                                    currentFromLanguage={currentFromLanguageState}
+                                                    onLanguageUpdate={(value) => {
+                                                        isUserChosenTargetLanguage = true
+                                                        updateTranslateTargetLanguage(value)
+                                                    }}
+                                                    currentTargetLanguage={translateTargetLanguage}
+                                                />
+                                            }
+                                        />
+                                    )
+                                })}
+                            </List.Section>
+                        )
+                    })}
+                </Fragment>
+            )
+        }
+
+        return (
+            <List.Item
+                title={`Sorry! We have some problems..`}
+                subtitle={`code: ${fetchResultStateCode}`}
+                icon={{ source: Icon.XmarkCircle, tintColor: Color.Red }}
+                actions={
+                    <ActionPanel>
+                        <Action.OpenInBrowser
+                            title="Help"
+                            icon={Icon.QuestionMark}
+                            url="https://github.com/Haojen/raycast-Parrot#error-code-information"
+                        />
+                    </ActionPanel>
+                }
+            />
+        )
+    }
+    function onInputChangeEvt(queryText: string) {
+        updateLoadingState(false)
+        clearTimeout(delayFetchTranslateAPITimer)
+
+        const text = queryText.trim()
+        if (text.length > 0) {
+            delayFetchTranslateAPITimer = setTimeout(() => {
+                updateCopyModeState(() => {
+                    const freshCopyModeState = detectIsUppercaseCopyOrLowerCaseCopy(text)
+
+                    const freshInputValue = removeDetectCopyModeSymbol(text, freshCopyModeState)
+                    updateInputState(freshInputValue)
+
+                    return freshCopyModeState
+                })
+            }, 800)
+            return
+        }
+
+        updateTranslateResultState([])
+    }
+
+    return (
+        <List
+            isLoading={isLoadingState}
+            searchBarPlaceholder={"Translate to"}
+            onSearchTextChange={onInputChangeEvt}
+            actions={
+                <ActionPanel>
+                    <ActionFeedback />
+                </ActionPanel>
+            }
+        >
+            <ListDetail />
```

Comment:
```suggestion
            <List.EmptyView
                icon="parrot-icon-small.png"
                title="Type something to get started"
            />
            <ListDetail />
```
Add the attached icon to assets 😄